frontend/src/components/Search/SearchForm/SearchForm.tsx

```diff
@@ -263,14 +262,11 @@ const SearchForm: React.FC<SearchFormProps> = ({
 									<Box display="flex" alignItems="center">
 										{loading ? (
 											<LoadingBox />
-										) : (
-											resultCount != null && (
-												<Text color="weak">
-													{formatNumber(resultCount)}{' '}
-													results
-												</Text>
-											)
-										)}
+										) : resultFormatted ? (
```

Comment:
Not sure this is the place for the comment, but noticed this text when 0 results - don't think it needs to block though

![Screenshot 2024-10-15 at 10 23 28 AM](https://github.com/user-attachments/assets/ab09b4da-e890-40eb-a05e-58cc4ba942d3)